### PR TITLE
Add show local sessions command

### PR DESCRIPTION
**Pull request by Aiee**
#### What type of PR is this?
- [ ] bug
- [x] feature
- [x] enhancement

#### What does this PR do?
1. Add `SHOW LOCAL SESSIONS` command to list the sessions of the current graph node
2. [**Incompatibility**] Deprecate `SHOW ALL QUERIES`. Now `SHOW QUERIES` lists all queries in the whole cluster and `SHOW LOCAL QUERIES` lists all queries in the current graph node.

#### Which issue(s)/PR(s) this PR relates to?
Depends on https://github.com/vesoft-inc/nebula/pull/3486 
Close https://github.com/vesoft-inc/nebula/issues/3547
#### Checklist:
- [x] Documentation affected (Please add the label if documentation needs to be modified.)
- [x] Incompatibility (If it breaks the compatibility, please describe it and add the corresponding label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
>                                                                 `

### CODE REVIEW

**Comment by jievince** [comment]
Plz resolve the conflicts

**Review by jievince — COMMENTED** [review]
LGTM

File: tests/job/test_session.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import time
 import pytest
 import concurrent
+import pdb
```
Comment (HarrisChu) [inline tests/job/test_session.py:12]: delete this

**Comment by HarrisChu** [comment]
incompatible

`SHOW QUERIES` -> `SHOW LOCAL QUERIES`
`SHOW ALL QUERIES` -> `SHOW QUERIES`

**Review by yixinglu — APPROVED** [review]
LGTM

**Review by jievince — APPROVED** [review]
LGTM

**Review by yixinglu — APPROVED** [review]
LGTM

**Pull request merged**